platformLibs/src/platform/windows/windows.def

```diff
@@ -3,5 +3,5 @@ headers = wtypes.h minwindef.h windows.h commctrl.h dwmapi.h shlobj.h shlwapi.h
     urlmon.h usp10.h uxtheme.h vfw.h wininet.h winsock2.h ws2tcpip.h ws2def.h
 compilerOpts = -DUNICODE -DWINVER=0x0601 -D_WIN32_WINNT=0x0601 -DWINAPI_FAMILY=3 \
     -Wno-incompatible-pointer-types -Wno-deprecated-declarations
-linkerOpts = -lcomctl32 -lcrypt32 -lshlwapi -lshell32 -limm32 -lusp10 -lwininet
+linkerOpts = -lcomctl32 -lcrypt32 -lshlwapi -lshell32 -limm32 -lusp10 -lwininet -lgdi32 -luser32 -lkernel32
```

Comment:
Maybe we'd better split windows platform libs in more finegrained way, as for now this one is getting pretty big and monolithic. Anyway, not in scope of this change.

Reply:
Agree. Though it may be a bit complicated:

For ChoosePixelFormat and SetPixelFormat they are defined in wingdi.h and exposed in gdi32.dll:
https://msdn.microsoft.com/en-us/library/windows/desktop/dd369049(v=vs.85).aspx

But for example wgl* is in wingdi.h, but exported in opengl32.dll:
https://msdn.microsoft.com/es-es/library/windows/desktop/dd374379(v=vs.85).aspx

And that's an example. I'm not sure how many things are defined in one header and exported in another dll.

One thing that would be awesome, would be to check how many/which functions are matching the klib with the provided libraries, to determine if we are missing libraries and knowing beforehand about linking errors.